src/views/List.jsx

```diff
@@ -7,11 +7,9 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
-				{/**
-				 * TODO: write some JavaScript that renders the `data` array
-				 * using the `ListItem` component that's imported at the top
-				 * of this file.
-				 */}
+				{data.map((item, index) => (
+					<ListItem name={item.name} key={index} />
```

Comment:
Suggestion: As the key, you could use the `item.id` instead of the index for more specificity, since it's available.